server.js

```diff
@@ -7,9 +7,30 @@ const getFiles = async ()=>{
     return [indexHtml,indexJs]
 }
 
-
 http.createServer(async ({ url }, res) => {
     const [indexHtml, indexJs] = await getFiles()
+
+    if (url.includes('value')) {
+        const inputValue = url.match(/=(\w+(?:%20\w+)*)/g)
```

Comment:
Do we need 'g' here? You are taking only the first element from the inputValue later on in the 18 line anyway.

Reply:
Removed ;)